backends/imgui_impl_metal.mm

```diff
@@ -283,8 +451,14 @@ - (void)makeFontTextureWithDevice:(id<MTLDevice>)device
                                                                                                 height:(NSUInteger)height
                                                                                              mipmapped:NO];
     textureDescriptor.usage = MTLTextureUsageShaderRead;
+    // Only override the storageMode for macOS with GPUs supporting unified memory,
+    // as the default value, per Apple docs, is already set correctly.
 #if TARGET_OS_OSX || TARGET_OS_MACCATALYST
-    textureDescriptor.storageMode = MTLStorageModeManaged;
+    if (@available(macOS 10.15, macCatalyst 13.0, *)) {
+        if (device.hasUnifiedMemory) {
```

Comment:
This does not build on 10.13 (Xcode 10.1). `macCatalyst` and `hasUnifiedMemory` symbols are not available.

Reply:
I'll add preprocessor checks for this.